### PR TITLE
Add parmeet9891 to Contributors list

### PR DESCRIPTION
**Pull request by parmeet9891**
I have added "ABC Warehouse" in the brands.

### CODE REVIEW

File: brands/shop/electronics.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
+   "shop/electronics|ABC_Warehouse": {
```
Comment (bhousel) [inline brands/shop/electronics.json]: Looks mostly good! - can you remove the underscore from the name here?
```suggestion
   "shop/electronics|ABC Warehouse": {
```

Comment (parmeet9891) [reply brands/shop/electronics.json]: @bhousel - I have removed underscore from the name. Thanks.

**Comment by bhousel** [comment]
Thanks @parmeet9891 for the quick fix, and welcome! 👋

**Comment by bhousel** [comment]
I also set the countryCode to `us` for this brand in 79f7528fe because Wikipedia says they "operate 42 stores in Michigan, Ohio and Indiana"

**Comment by parmeet9891** [comment]
Cool!

**Pull request merged**